man/bundle-doctor.ronn

```diff
@@ -0,0 +1,18 @@
+bundle-doctor(1) -- Checks for missing OS dependencies
```

Comment:
I would just say `Checks the bundle for common problems` because `bundler doctor` is intended to do more than just checking for OS dependencies.

Reply:
Gotcha. I'll read up on `bundle doctor` and flesh the description out a bit.